multi-bracket-validation/multi-bracket-validation.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+import Stack from './lib/stack';
+
```

Comment:
Looks great, good job using a stack for this problem, which is exactly the best way to solve it.